panther_lights_controller/src/controller_node.py

```diff
@@ -251,6 +262,24 @@ def _get_image_animation_description(self, animation: LEDImageAnimation):
 
         return animation_description
 
+    def _update_animations(self):
+
+        user_animations = rospy.get_param('~user_animations', '')
+
+        for animation in user_animations:
+            # ID numbers from 0 to 20 are reserved for system animations
+            if animation['id'] > 20:
```

Comment:
ID is not checked if it is positive value

Reply:
If ID is greater than 20 then it is positive

Reply:
IF you put it this way then yes. It is